process/sp.py

```diff
@@ -141,7 +161,7 @@
 
     # read sample points from disk (in city-specific geopackage)
     samplePointsData = gpd.read_file(
-        gpkgPath, layer=config["parameters"]["samplePoints"])
```

Comment:
There appears to be some inconsistency throughout with using either single quotes `'` vs double quotes `"`... I'd suggest standardizing around single quotes.